node/grpc/server.go

```diff
@@ -283,7 +283,10 @@ func (s *Server) AttestBatch(ctx context.Context, in *pb.AttestBatchRequest) (*p
 	if err != nil {
 		return nil, fmt.Errorf("failed to get the batch header hash: %w", err)
 	}
-	sig := s.node.KeyPair.SignMessage(batchHeaderHash)
+	sig, err := s.node.GetSignature(ctx, batchHeaderHash)
```

Comment:
consider renaming `GetSignature` to just `Sign`?

Reply:
yea - good suggestion

Reply:
40777795d4f4d89cea862f8d8a7f5115e12af6dd